networkx/generators/classic.py

```diff
@@ -509,7 +509,7 @@ def dorogovtsev_goltsev_mendes_graph(n, create_using=None):
         The generation number.
 
     create_using : NetworkX graph constructor, optional (default=nx.Graph)
-        Graph type to create. If graph instance, then cleared before populated.
+        Graph type to create. Directed graphs and multigraphs are not supported.
```

Comment:
I'm assuming this is related to your comment on another PR that we shouldn't encourage giving `create_using` a graph instance (which I think is a good thing!). Will this eventually be deprecated?

Reply:
Also, I'm not totally convinced on duplicating information in the docstrings, and this is already mentioned at least once in the "Raises" section... You probably know best though!

Reply:
> Will this eventually be deprecated?

AFAIK it hasn't been discussed, but I'd personally be against deprecating it given the potential disruption to existing code for arguably very little benefit.

> I'm not totally convinced on duplicating information in the docstrings, and this is already mentioned at least once in the "Raises" section

Yeah it's admittedly quite subjective - on the one hand, I certainly agree with the "one source of truth" argument; OTOH I think there's value to having the restrictions clearly stated in the parameter description, especially when it's short.

In this particular case, the [rendered docstring](https://networkx.org/documentation/latest/reference/generated/networkx.generators.classic.dorogovtsev_goltsev_mendes_graph.html#networkx.generators.classic.dorogovtsev_goltsev_mendes_graph) is so short that it really doesn't make that much of a difference at all!